src/nodejs/neon-js/modules/getBlock.js

```diff
@@ -0,0 +1,85 @@
+// rpc getBlock module
+// This is called by cli/rpc/getBlock CLI wrapper
+// Main Dependency: neon-js
+// This returns a block
+
+require('module-alias/register')
+
+const _       = require('underscore')
+const neon    = require('@cityofzion/neon-js')
+const dbg     = require('nodejs_util/debug')
+
+// Pass an object named config of the following format to control module behavior
+// program.debug    // Toggle debugging
+// program.node     // Set RPC node to use (be sure to preface with https://)
+// program.hash     // Specify the hash of the block to fetch, if no hash or index is supplied will get the tallest
+// program.time     // Only return the time field of the last block
+// program.human    // Make dates human-readable
+// program.index    // specify the number of the block to fetch, if no hash or index is supplied will get the tallest
+
+
+exports.run = (config) => {
+  let program = {}
+  let defly = false
+
+  if (config) program = config
+  else {
+    program.debug = false
+    program.node = ''
+    program.hash = null
+    program.time = false
+    program.human = false
+    program.index = 0
+  }
+
+  function print(msg) {
+    console.log(msg);
```

Comment:
It feels a little redundant to have a function for 1 console log line :P

Reply:
Noted, but I'd like to keep the print macro for now.